front/src/applications/stdcmV2/components/StdcmConfig.tsx

```diff
@@ -117,6 +124,14 @@ const StdcmConfig = ({
     }
   }, [isDebugMode]);
 
+  useEffect(() => {
+    if (!infra || infra.state === 'CACHED') {
```

Comment:
If we have no infra, it's because the first call to the backend is not finished. The first call will send the infra, even if it is not loaded.
This condition avoid a flash on refresh:
[Capture vidéo du 2024-10-11 17-14-09.webm](https://github.com/user-attachments/assets/1c1b18f0-6fc9-411a-af66-fb7921675085)